queue_test.go

```diff
@@ -20,12 +20,12 @@ var _ = Describe("Queue", func() {
 	Describe("Push", func() {
 		var operations []*fake_operationq.FakeOperation
 
-		JustBeforeEach(func(done Done) {
-			for _, o := range operations {
-				queue.Push(o)
-			}
-
-			close(done)
+		JustBeforeEach(func() {
+			go func() {
+				for _, o := range operations {
+					queue.Push(o)
+				}
+			}()
```

Comment:
Sorry for the late review, but any chance you remember why this logic got stuck in a goroutine instead of running it inline?